lib/validators/calculating_smoking_gun_validator.rb

```diff
@@ -0,0 +1,82 @@
+module Validators
+  class CalculatingSmokingGunValidator < SmokingGunValidator
+    include Validators::Validator
+    def initialize(measures, records, test_id)
+      if measures.length > 1
+        # throw some kind of an error here because we only want to do this for
```

Comment:
Is there supposed to be an actual error thrown here, or is this just left as a hook for the UI people?